shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Utxow.hs

```diff
@@ -310,9 +321,28 @@ utxoWitnessed =
         TRC (UtxoEnv slot pp stakepools genDelegs, u, tx)
 
 instance
```

Comment:
It seems that each constraint on the `STS` instance of `UTXOW` is needed here too. I haven't look at `Embed` closely, but it's kind of strange that we really need *all* of these constraints just to embed one error into another.

Reply:
`Embed` has a superclass constraint of `STS`